src/frontend/metrics/deviceDiagnosticMetrics.ts

```diff
@@ -0,0 +1,68 @@
+import pProps from 'p-props';
+import {Dimensions, PixelRatio, Platform} from 'react-native';
+import DeviceInfo from 'react-native-device-info';
```

Comment:
Just highlighting that [expo-device](https://docs.expo.dev/versions/latest/sdk/device/) does, I think, more normalization of this data between platforms, and also provides more data as constants rather than async. We are however already using react-native-device-info in a few places, so fine to use here for now. As we add the daily app data, it may be more convenient to switch to expo-device.

Reply:
The fact that it returns data synchronously makes things much easier. Switched in a5b55f246c9a7f5eecd2ace1fdeb2ceb25d986e8.